### ISSUE TITLE
XDG compliance

### ISSUE THREAD

**Issue by hseg** [haskell/haskeline#150]
Please prioritize `getXdgDirectory XdgConfig	"haskeline/haskeline"` over `getHomeDirectory <&> (</> ".haskeline")` as a config location. This behaviour is supported since `directory-1.2.3.0`, released in 2015-07.
cf. <https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html>

Otherwise, please do what readline does and provide an analogue of `INPUTRC`.
Of the two options, the former looks simpler.

**Comment by eankeen** [comment]
Yes, I am for this as well. I was looking for an environment variable and concluded there was none after seeing `readPrefsFromHome` in `System/Console/Haskeline/InputT.hs` Direct compliance is preferred in my opinion, but as long as there is an option to change, that would be fantastic

**Comment by athas** [comment]
Solved in #157.
